pkg/infra/grn/grn_test.go

```diff
@@ -1,66 +1,70 @@
-package grn
+package grn_test
```

Comment:
@mildwonkey -- i updated this to be the test package so we see how it feels to use:
```
grn := grn.GRN{...}
```
not a big deal, but nice to exercise the public flavor

Reply:
Ah ok. I'm not generally a fan of the  `_test` pattern without very solid reasoning; that doesn't feel like reason enough to me (it's not functionally different from `grn := GRN{...}`). On the other hand this doesn't really matter unless we needed to test internal functions or interactions with other packages is needed, and it's tiny package, so I won't fight you on it